src/__tests__/Dropdown.spec.js

```diff
@@ -452,7 +452,7 @@ describe('Dropdown', () => {
 
       expect(Dropdown.prototype.toggle.mock.calls.length).toBe(0);
 
-      wrapper.find('#first').hostNodes().simulate('keydown', { which: 52 });
+      wrapper.find('#dropdown').hostNodes().simulate('keydown', { which: 52 });
```

Comment:
This test doesn't work when we trigger `keydown` events directly on the dropdown items, but I'm unable to actually replicate that scenario from my browser. It appears to me like this test is a more accurate representation of a user interaction. And it makes the tests pass. Of course, I could be wrong! 😄